src/V3Const.cpp

```diff
@@ -480,9 +480,9 @@ class ConstBitOpTreeVisitor final : public AstNVisitor {
             incrOps(andp, __LINE__);
 
             // Mark all bits checked in this reduction
-            const int maxBitIdx = std::min(ref.m_lsb + maskNum.width(), ref.width());
-            for (int bitIdx = ref.m_lsb; bitIdx < maxBitIdx; ++bitIdx) {
+            for (int bitIdx = ref.m_lsb; bitIdx < ref.width(); ++bitIdx) {
                 const int maskIdx = bitIdx - ref.m_lsb;
+                if (maskIdx >= maskNum.width()) break;
```

Comment:
To me this seems to do exactly the same thing as the old version, so maybe it does not need changing? 

`maskIdx` ranges over `[0, maskNum.width()-1]` or `[0, ref.width() - 1 - ref.m_lsb]`, whichever is shorter, same as before?